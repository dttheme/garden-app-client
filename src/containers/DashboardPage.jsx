import React, { Component } from "react";
import { Field, reduxForm } from 'redux-form';
import { connect } from 'react-redux';
import { createPlant } from '../actions/index.actions';


// import './Dashboard.css';

export class DashboardPage extends Component {
  submitHandler(val) {
    const plantName = val.plantName;
    const plantDate = val.plantDate;
    const numberPlanted = val.numberPlanted;
    const plantLocation = val.plantLocation;
    const waterFrequency = val.waterFrequency;
    this.props.createPlant(plantName, plantDate, numberPlanted, plantLocation, waterFrequency)
  }
  render() {
    return (
      <div className="ml7 mr7 ba bw2 pa3">
        <h2 className="f2 h3 tc">
          Dashboard
        </h2>
        <form onSubmit={this.props.handleSubmit(this.submitHandler.bind(this))} method="POST">
          <label
            htmlFor="firstName"
            className="f3 fw5"
            >
            What kind of plant?
          </label>
          <Field
            name="plantName"
            component="input"
            type="text"
            placeholder="Plant Name"
            className="db mb4 mt2"
            />
          <label
            htmlFor="plantDate"
            className="f3 fw5"
            >
            When was it planted?
          </label>
          <Field
            name="plantDate"
            component="input"
            type="text"
            placeholder="Date Planted"
            className="db mb4 mt2"
            />
          <label
            htmlFor="numberPlanted"
            className="f3 fw5"
            >
            How many were planted?
          </label>
          <Field
            name="numberPlanted"
            component="input"
            type="number"
            value={1}
            min={1}
            max={100}
            className="db mb4 mt2"
            />
          <label
            htmlFor="plantLocation"
            className="f3 fw5"
            >
            Where is it planted?</label>
          <Field
            name="plantLocation"
            component="input"
            type="text"
            placeholder="Plant Location"
            className="db mb4 mt2"
            />
          <label
            htmlFor="waterFrequency"
            className="f3 fw5"
            >
            How often should you water?
          </label>
          <label>
            <Field
              name="waterFrequency"
              component="input"
              type="radio"
              value="Once a day"
              /> {' '} Once a day
            </label>
            <label>
              <Field
                name="waterFrequency"
                component="input"
                type="radio"
                value="Once a week"
                /> {' '} Once a week
              </label>
              <label>
                <Field
                  name="waterFrequency"
                  component="input"
                  type="radio"
                  value="Once a month"
                  /> {' '} Once a month
                </label>
                <button type="submit" className="db">
                  Submit
                </button>
              </form>
            </div>
          );
        }
      }

      const mapStateToProps = (state) => ({

      });

export default reduxForm({
  form: "newPlant",
})(connect(mapStateToProps, {createPlant})(DashboardPage));
