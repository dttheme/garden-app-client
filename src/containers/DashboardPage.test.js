import React from 'react';
import { shallow, mount } from 'enzyme';
import { DashboardPage } from './DashboardPage';

describe('<DashboardPage />', () => {

  it('should render page', () => {
    const wrapper = shallow(<DashboardPage handleSubmit={() => {}} />);
    expect(wrapper).toMatchSnapshot()
  });
});
