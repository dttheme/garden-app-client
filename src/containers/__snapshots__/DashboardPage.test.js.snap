// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardPage /> should render page 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DashboardPage
    handleSubmit={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="f2 tracked"
>
          Dashboard
</h2>,
        <form
          method="POST"
          onSubmit={undefined}
>
          <label
                    className="tracked"
                    htmlFor="firstName"
          >
                    What kind of plant?
          </label>
          <Field
                    className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                    component="input"
                    name="plantName"
                    placeholder="Tomato"
                    required={true}
                    type="text"
          />
          <label
                    className="tracked"
                    htmlFor="plantDate"
          >
                    When was it planted?
          </label>
          <Field
                    className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                    component="input"
                    name="plantDate"
                    placeholder="Late February"
                    required={true}
                    type="text"
          />
          <label
                    className="tracked"
                    htmlFor="numberPlanted"
          >
                    How many were planted?
          </label>
          <Field
                    className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                    component="input"
                    max={100}
                    min={1}
                    name="numberPlanted"
                    type="number"
                    value={1}
          />
          <label
                    className="tracked"
                    htmlFor="plantLocation"
          >
                    Where is it planted?
          </label>
          <Field
                    className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                    component="input"
                    name="plantLocation"
                    placeholder="Front porch"
                    required={true}
                    type="text"
          />
          <label
                    className="tracked"
                    htmlFor="waterFrequency"
          >
                    How often should you water?
          </label>
          <label
                    className="db"
          >
                    <Field
                              component="input"
                              name="waterFrequency"
                              type="radio"
                              value="Once a day"
                    />
                     
                     
                     Once a day
          </label>
          <label
                    className="db"
          >
                    <Field
                              component="input"
                              name="waterFrequency"
                              type="radio"
                              value="Once a week"
                    />
                     
                     
                     Once a week
          </label>
          <label
                    className="db"
          >
                    <Field
                              component="input"
                              name="waterFrequency"
                              type="radio"
                              value="Once a month"
                    />
                     
                     
                     Once a month
          </label>
          <button
                    className="w-90 w-70-m w-50-ns f4 link dim ba bw2 ph3 pv2 mt3 mb2 dib near-black"
                    type="submit"
          >
                    Submit
          </button>
</form>,
      ],
      "className": "dark-gray bg-dark-green w-100 w-80-m w-60-ns mt5 mt7-m mt6-ns mb5 ml7-ns ml5-m ba-ns bw3-ns b--green pl6-ns pl4-m pl4 pb4",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Dashboard",
          "className": "f2 tracked",
        },
        "ref": null,
        "rendered": "Dashboard",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <label
              className="tracked"
              htmlFor="firstName"
>
              What kind of plant?
</label>,
            <Field
              className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
              component="input"
              name="plantName"
              placeholder="Tomato"
              required={true}
              type="text"
/>,
            <label
              className="tracked"
              htmlFor="plantDate"
>
              When was it planted?
</label>,
            <Field
              className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
              component="input"
              name="plantDate"
              placeholder="Late February"
              required={true}
              type="text"
/>,
            <label
              className="tracked"
              htmlFor="numberPlanted"
>
              How many were planted?
</label>,
            <Field
              className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
              component="input"
              max={100}
              min={1}
              name="numberPlanted"
              type="number"
              value={1}
/>,
            <label
              className="tracked"
              htmlFor="plantLocation"
>
              Where is it planted?
</label>,
            <Field
              className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
              component="input"
              name="plantLocation"
              placeholder="Front porch"
              required={true}
              type="text"
/>,
            <label
              className="tracked"
              htmlFor="waterFrequency"
>
              How often should you water?
</label>,
            <label
              className="db"
>
              <Field
                            component="input"
                            name="waterFrequency"
                            type="radio"
                            value="Once a day"
              />
               
               
               Once a day
</label>,
            <label
              className="db"
>
              <Field
                            component="input"
                            name="waterFrequency"
                            type="radio"
                            value="Once a week"
              />
               
               
               Once a week
</label>,
            <label
              className="db"
>
              <Field
                            component="input"
                            name="waterFrequency"
                            type="radio"
                            value="Once a month"
              />
               
               
               Once a month
</label>,
            <button
              className="w-90 w-70-m w-50-ns f4 link dim ba bw2 ph3 pv2 mt3 mb2 dib near-black"
              type="submit"
>
              Submit
</button>,
          ],
          "method": "POST",
          "onSubmit": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "What kind of plant?",
              "className": "tracked",
              "htmlFor": "firstName",
            },
            "ref": null,
            "rendered": "What kind of plant?",
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "className": "db mb4 mt2 w-90 w-70-m w-50-ns pa2",
              "component": "input",
              "name": "plantName",
              "placeholder": "Tomato",
              "required": true,
              "type": "text",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "When was it planted?",
              "className": "tracked",
              "htmlFor": "plantDate",
            },
            "ref": null,
            "rendered": "When was it planted?",
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "className": "db mb4 mt2 w-90 w-70-m w-50-ns pa2",
              "component": "input",
              "name": "plantDate",
              "placeholder": "Late February",
              "required": true,
              "type": "text",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "How many were planted?",
              "className": "tracked",
              "htmlFor": "numberPlanted",
            },
            "ref": null,
            "rendered": "How many were planted?",
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "className": "db mb4 mt2 w-90 w-70-m w-50-ns pa2",
              "component": "input",
              "max": 100,
              "min": 1,
              "name": "numberPlanted",
              "type": "number",
              "value": 1,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Where is it planted?",
              "className": "tracked",
              "htmlFor": "plantLocation",
            },
            "ref": null,
            "rendered": "Where is it planted?",
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "className": "db mb4 mt2 w-90 w-70-m w-50-ns pa2",
              "component": "input",
              "name": "plantLocation",
              "placeholder": "Front porch",
              "required": true,
              "type": "text",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "How often should you water?",
              "className": "tracked",
              "htmlFor": "waterFrequency",
            },
            "ref": null,
            "rendered": "How often should you water?",
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Field
                  component="input"
                  name="waterFrequency"
                  type="radio"
                  value="Once a day"
/>,
                " ",
                " ",
                " Once a day",
              ],
              "className": "db",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "component": "input",
                  "name": "waterFrequency",
                  "type": "radio",
                  "value": "Once a day",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              " ",
              " ",
              " Once a day",
            ],
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Field
                  component="input"
                  name="waterFrequency"
                  type="radio"
                  value="Once a week"
/>,
                " ",
                " ",
                " Once a week",
              ],
              "className": "db",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "component": "input",
                  "name": "waterFrequency",
                  "type": "radio",
                  "value": "Once a week",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              " ",
              " ",
              " Once a week",
            ],
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Field
                  component="input"
                  name="waterFrequency"
                  type="radio"
                  value="Once a month"
/>,
                " ",
                " ",
                " Once a month",
              ],
              "className": "db",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "component": "input",
                  "name": "waterFrequency",
                  "type": "radio",
                  "value": "Once a month",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              " ",
              " ",
              " Once a month",
            ],
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Submit",
              "className": "w-90 w-70-m w-50-ns f4 link dim ba bw2 ph3 pv2 mt3 mb2 dib near-black",
              "type": "submit",
            },
            "ref": null,
            "rendered": "Submit",
            "type": "button",
          },
        ],
        "type": "form",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="f2 tracked"
>
            Dashboard
</h2>,
          <form
            method="POST"
            onSubmit={undefined}
>
            <label
                        className="tracked"
                        htmlFor="firstName"
            >
                        What kind of plant?
            </label>
            <Field
                        className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                        component="input"
                        name="plantName"
                        placeholder="Tomato"
                        required={true}
                        type="text"
            />
            <label
                        className="tracked"
                        htmlFor="plantDate"
            >
                        When was it planted?
            </label>
            <Field
                        className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                        component="input"
                        name="plantDate"
                        placeholder="Late February"
                        required={true}
                        type="text"
            />
            <label
                        className="tracked"
                        htmlFor="numberPlanted"
            >
                        How many were planted?
            </label>
            <Field
                        className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                        component="input"
                        max={100}
                        min={1}
                        name="numberPlanted"
                        type="number"
                        value={1}
            />
            <label
                        className="tracked"
                        htmlFor="plantLocation"
            >
                        Where is it planted?
            </label>
            <Field
                        className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                        component="input"
                        name="plantLocation"
                        placeholder="Front porch"
                        required={true}
                        type="text"
            />
            <label
                        className="tracked"
                        htmlFor="waterFrequency"
            >
                        How often should you water?
            </label>
            <label
                        className="db"
            >
                        <Field
                                    component="input"
                                    name="waterFrequency"
                                    type="radio"
                                    value="Once a day"
                        />
                         
                         
                         Once a day
            </label>
            <label
                        className="db"
            >
                        <Field
                                    component="input"
                                    name="waterFrequency"
                                    type="radio"
                                    value="Once a week"
                        />
                         
                         
                         Once a week
            </label>
            <label
                        className="db"
            >
                        <Field
                                    component="input"
                                    name="waterFrequency"
                                    type="radio"
                                    value="Once a month"
                        />
                         
                         
                         Once a month
            </label>
            <button
                        className="w-90 w-70-m w-50-ns f4 link dim ba bw2 ph3 pv2 mt3 mb2 dib near-black"
                        type="submit"
            >
                        Submit
            </button>
</form>,
        ],
        "className": "dark-gray bg-dark-green w-100 w-80-m w-60-ns mt5 mt7-m mt6-ns mb5 ml7-ns ml5-m ba-ns bw3-ns b--green pl6-ns pl4-m pl4 pb4",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Dashboard",
            "className": "f2 tracked",
          },
          "ref": null,
          "rendered": "Dashboard",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <label
                className="tracked"
                htmlFor="firstName"
>
                What kind of plant?
</label>,
              <Field
                className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                component="input"
                name="plantName"
                placeholder="Tomato"
                required={true}
                type="text"
/>,
              <label
                className="tracked"
                htmlFor="plantDate"
>
                When was it planted?
</label>,
              <Field
                className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                component="input"
                name="plantDate"
                placeholder="Late February"
                required={true}
                type="text"
/>,
              <label
                className="tracked"
                htmlFor="numberPlanted"
>
                How many were planted?
</label>,
              <Field
                className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                component="input"
                max={100}
                min={1}
                name="numberPlanted"
                type="number"
                value={1}
/>,
              <label
                className="tracked"
                htmlFor="plantLocation"
>
                Where is it planted?
</label>,
              <Field
                className="db mb4 mt2 w-90 w-70-m w-50-ns pa2"
                component="input"
                name="plantLocation"
                placeholder="Front porch"
                required={true}
                type="text"
/>,
              <label
                className="tracked"
                htmlFor="waterFrequency"
>
                How often should you water?
</label>,
              <label
                className="db"
>
                <Field
                                component="input"
                                name="waterFrequency"
                                type="radio"
                                value="Once a day"
                />
                 
                 
                 Once a day
</label>,
              <label
                className="db"
>
                <Field
                                component="input"
                                name="waterFrequency"
                                type="radio"
                                value="Once a week"
                />
                 
                 
                 Once a week
</label>,
              <label
                className="db"
>
                <Field
                                component="input"
                                name="waterFrequency"
                                type="radio"
                                value="Once a month"
                />
                 
                 
                 Once a month
</label>,
              <button
                className="w-90 w-70-m w-50-ns f4 link dim ba bw2 ph3 pv2 mt3 mb2 dib near-black"
                type="submit"
>
                Submit
</button>,
            ],
            "method": "POST",
            "onSubmit": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "What kind of plant?",
                "className": "tracked",
                "htmlFor": "firstName",
              },
              "ref": null,
              "rendered": "What kind of plant?",
              "type": "label",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "className": "db mb4 mt2 w-90 w-70-m w-50-ns pa2",
                "component": "input",
                "name": "plantName",
                "placeholder": "Tomato",
                "required": true,
                "type": "text",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "When was it planted?",
                "className": "tracked",
                "htmlFor": "plantDate",
              },
              "ref": null,
              "rendered": "When was it planted?",
              "type": "label",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "className": "db mb4 mt2 w-90 w-70-m w-50-ns pa2",
                "component": "input",
                "name": "plantDate",
                "placeholder": "Late February",
                "required": true,
                "type": "text",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "How many were planted?",
                "className": "tracked",
                "htmlFor": "numberPlanted",
              },
              "ref": null,
              "rendered": "How many were planted?",
              "type": "label",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "className": "db mb4 mt2 w-90 w-70-m w-50-ns pa2",
                "component": "input",
                "max": 100,
                "min": 1,
                "name": "numberPlanted",
                "type": "number",
                "value": 1,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Where is it planted?",
                "className": "tracked",
                "htmlFor": "plantLocation",
              },
              "ref": null,
              "rendered": "Where is it planted?",
              "type": "label",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "className": "db mb4 mt2 w-90 w-70-m w-50-ns pa2",
                "component": "input",
                "name": "plantLocation",
                "placeholder": "Front porch",
                "required": true,
                "type": "text",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "How often should you water?",
                "className": "tracked",
                "htmlFor": "waterFrequency",
              },
              "ref": null,
              "rendered": "How often should you water?",
              "type": "label",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Field
                    component="input"
                    name="waterFrequency"
                    type="radio"
                    value="Once a day"
/>,
                  " ",
                  " ",
                  " Once a day",
                ],
                "className": "db",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "component": "input",
                    "name": "waterFrequency",
                    "type": "radio",
                    "value": "Once a day",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                " ",
                " ",
                " Once a day",
              ],
              "type": "label",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Field
                    component="input"
                    name="waterFrequency"
                    type="radio"
                    value="Once a week"
/>,
                  " ",
                  " ",
                  " Once a week",
                ],
                "className": "db",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "component": "input",
                    "name": "waterFrequency",
                    "type": "radio",
                    "value": "Once a week",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                " ",
                " ",
                " Once a week",
              ],
              "type": "label",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Field
                    component="input"
                    name="waterFrequency"
                    type="radio"
                    value="Once a month"
/>,
                  " ",
                  " ",
                  " Once a month",
                ],
                "className": "db",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "component": "input",
                    "name": "waterFrequency",
                    "type": "radio",
                    "value": "Once a month",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                " ",
                " ",
                " Once a month",
              ],
              "type": "label",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Submit",
                "className": "w-90 w-70-m w-50-ns f4 link dim ba bw2 ph3 pv2 mt3 mb2 dib near-black",
                "type": "submit",
              },
              "ref": null,
              "rendered": "Submit",
              "type": "button",
            },
          ],
          "type": "form",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
